src/content/snippets/@emotion/styled/Header.js.mdx

```diff
@@ -0,0 +1,74 @@
+```js
+// ./src/stories/Header.js
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+
+import { Button } from './Button';
+import './header.css';
```

Comment:
Is this still used?

Reply:
Yes, just to avoid needing to rewrite the entire component with the same styles